speech_assets/sampleUtterances.txt

```diff
@@ -28,9 +28,9 @@ GeeMusicPlayIFLRadioIntent Play I'm feeling lucky station
 GeeMusicPlayIFLRadioIntent Play my radio station
 GeeMusicPlayIFLRadioIntent Play my station
 
-GeeMusicCurrentlyPlayingIntent What is the current song?
-GeeMusicCurrentlyPlayingIntent What is currently playing?
-GeeMusicCurrentlyPlayingIntent What is playing right now?
-GeeMusicCurrentlyPlayingIntent What song is this?
-GeeMusicCurrentlyPlayingIntent What is the name of this song?
-GeeMusicCurrentlyPlayingIntent What is this?
+GeeMusicCurrentlyPlayingIntent What is the current song
```

Comment:
When the sampleUtterances are copied into the interaction module it gives you an error
```
Error: There was a problem with your request: The token 'song?' is not valid. Each token must consist only of unicode characters, spaces and valid punctuation marks. Valid punctuation marks are periods for abbreviations, underscores, possessive apostrophes, and hyphens. Occurred in sample 'GeeMusicCurrentlyPlayingIntent What is the current song?' on line 45.
```